custom-targets/infrastructure-manager/README.md

```diff
@@ -0,0 +1,75 @@
+# Cloud Deploy Infrastructure Manager Terraform Deployer Sample
+This directory contains a sample implementation of a Cloud Deploy Custom Target for deploying Google
+Cloud infrastructure resources via Terraform using [Infrastructure Manager](https://cloud.google.com/infrastructure-manager).
+
+**This is not an officially supported Google product, and it is not covered by a
+Google Cloud support contract. To report bugs or request features in a Google
+Cloud product, please contact [Google Cloud
+support](https://cloud.google.com/support).**
+
+# Quickstart
+
+A quickstart that uses this sample is available [here](./quickstart/QUICKSTART.md).
+
+# Configuration
+
+## Terraform Configuration
+The Terraform configuration provided when creating a Cloud Deploy Release must meet the [Infrastructure Manager
+constraints](https://cloud.google.com/infrastructure-manager/docs/terraform#constraints_on_terraform_configurations).
+
+> [!IMPORTANT] 
+> To ensure consistent Cloud Deploy Rollouts for a Release - remotely sourced modules should be [selected by revision](https://developer.hashicorp.com/terraform/language/modules/sources#selecting-a-revision), e.g. for a module in a Git repo specify a commit using its SHA-1 hash.
+
+## Deploy Parameters
+
+| Parameter | Required | Description |
+| --- | --- | --- |
+| customTarget/imProject | Yes | The project ID for the Infrastructure Manager Deployment |
+| customTarget/imLocation | Yes | The location for the Infrastructure Manager Deployment. See Infrastructure Manager supported locations [here](https://cloud.google.com/infrastructure-manager/docs/locations) |
```

Comment:
nit: I wounder if we should just default this to the CD project if not provided

Reply:
That's an interesting idea, will revisit since moving from required to optional should be fine later.